src/drawing/backend_impl/bitmap.rs

```diff
@@ -231,7 +231,7 @@ pub trait PixelFormat: Sized {
                         *buf.get_unchecked_mut(base + idx) = Self::byte_at(r, g, b, 0, idx);
                     }
                 } else {
-                    let alpha = alpha.min(1.0).max(0.0);
+                    let alpha = alpha.min(255.0 / 256.0).max(0.0);
```

Comment:
I believe we can juse use if alpha >= 255.0 / 256.0 as the check for if we need to directly override the pixel.

Reply:
Could you please elaborate? The then-branch above has the check for overriding the pixel (alpha >= 1.0), do I miss something here?

Reply:
I think the reason why it can overflow is because `(alpha * 256.0).floor()` returns a 256.0, this might be possible since we have floating point issues. But if you dig into the logic at this point, we actually split the range (0.0..1.0] to 256 different alpha levels, and which means anything is greater than 255.0 / 256.0 can be treated as directly overriding, right? 
By doing so, I believe there won't be any case causing overflow anymore.

Please let me know if I was wrong.

Reply:
Sorry, looks like I was not clear in the original PR message - the overflow is not happening here, but rather in the `animation` example. The fix is required to properly address invalid `alpha` values. I've added `test_draw_pixel_alphas` tests which you can use to verify the behaviour - without the fix, the bitmap code is crashing.